jenkins/job-configs/kubernetes-jenkins/bootstrap-ci-soak.yaml

```diff
@@ -61,7 +61,7 @@
     - kubernetes-soak-gce-federation-deploy:
         blocker: ci-kubernetes-soak-gce-federation-test
         job-name: ci-kubernetes-soak-gce-federation-deploy
-        frequency: 'H 0 * * 2'
+        frequency: 'H 0 * * *'
```

Comment:
@madhusudancs, this is so that we can speed up fixing the deployment if it happens everyday instead of weekly once. we shall change it back to 1 week after the build succeeds.